docker-compose.yml

```diff
@@ -40,6 +40,33 @@ services:
     volumes:
       - .:/app
 
+  wordpress_e2e_tests:
+    image: wordpress
+    ports:
+      - 8889:80
+    environment:
+      WORDPRESS_DB_NAME: wordpress_e2e_tests
+      WORDPRESS_DB_PASSWORD: example
+      ABSPATH: /usr/src/wordpress/
+    volumes:
+      - wordpress_e2e_tests:/var/www/html
+      - .:/var/www/html/wp-content/plugins/gutenberg
+      - ./test/e2e/test-plugins:/var/www/html/wp-content/plugins/gutenberg-test-plugins
+      - ./test/e2e/test-mu-plugins:/var/www/html/wp-content/mu-plugins
+
+  mysql_e2e_tests:
```

Comment:
I'm as excited as anyone about containers, but do we really need a seperate MySQL container here? Does it provide any isolation benefit that using a different `DB_NAME` doesn't give?

Reply:
Is that a lot of work (mostly time wise) to create a new database per one tests run when booting tests?

Or use some WP-CLI incantation to restore it to a fresh install? In general, we are using WP Admin UI to do some things which WP-CLI, in theory, could do with one line, like activating/disabling plugins. I have no idea if it is easy to integrate, but I thought I ask :)

Reply:
It takes a few seconds to drop the database and re-install WordPress, I think the slowest part is Docker taking a snapshot of the cli box when the command runs.

I'm more concerned about having two MySQL servers running at the same, chewing up RAM.

Reply:
I agree.

I thought I mentioned it... maybe it's removed in a new commit, but yeah I would prefer a single DB too. I think the Docker container itself couldn't do one, but we should be able to easily create a new database used for the test runs on the existing DB/container.

I'll change this up to use the existing DB. I wanted to use the same WP-CLI container as well but we'd need to install stuff in WordPress to make RESTful WP-CLI commands work if I understand it correctly and the CLI container is lightweight enough. 🤷‍♂️